Translations update from Fedora Weblate

weblate opened the pull request:
Translations update from [Fedora Weblate](https://translate.fedoraproject.org) for [fedora-infra/noggin](https://translate.fedoraproject.org/projects/fedora-infra/noggin/).



Current translation status:

![Weblate translation status](https://translate.fedoraproject.org/widget/fedora-infra/noggin/horizontal-auto.svg)

abompard commented:
Hey @cverna, it looks like you're the maintainer of the project on Weblate: https://translate.fedoraproject.org/projects/fedora-infra/noggin/
Could you please set it to add Signed-Off-By in the commits, or add me as an admin so I can do it?
Thanks!

yurchor committed:
Translated using Weblate (Ukrainian)

Currently translated at 99.3% (294 of 296 strings)

Co-authored-by: Yuri Chornoivan <yurchor@ukr.net>
Signed-off-by: Yuri Chornoivan <yurchor@ukr.net>
Translate-URL: https://translate.fedoraproject.org/projects/fedora-infra/noggin/uk/
Translation: fedora-infra/noggin

Commit:
Translated using Weblate (Finnish)

Currently translated at 75.3% (223 of 296 strings)

Co-authored-by: Linus Virtanen <linus.virtanen@protonmail.com>
Signed-off-by: Linus Virtanen <linus.virtanen@protonmail.com>
Translate-URL: https://translate.fedoraproject.org/projects/fedora-infra/noggin/fi/
Translation: fedora-infra/noggin

bordik committed:
Translated using Weblate (Ukrainian)

Currently translated at 99.3% (294 of 296 strings)

Translated using Weblate (Ukrainian)

Currently translated at 77.0% (228 of 296 strings)

Co-authored-by: Borys Dikovets <bordik@gmail.com>
Signed-off-by: Borys Dikovets <bordik@gmail.com>
Translate-URL: https://translate.fedoraproject.org/projects/fedora-infra/noggin/uk/
Translation: fedora-infra/noggin

kapatych-be committed:
Added translation using Weblate (Belarusian)

Co-authored-by: Maksim Kliazovich <maxklezovich@gmail.com>

cverna commented:
Sorry just noticed this, I just sent you the invite to take on the admin permissions.
